lib/livebook_web/live/home_live/session_list_component.ex

```diff
@@ -12,10 +12,16 @@ defmodule LivebookWeb.HomeLive.SessionListComponent do
 
     sessions = sort_sessions(sessions, socket.assigns.order_by)
 
+    show_autosave_note? =
+      case Livebook.Config.autosave_path() do
+        nil -> false
+        path -> File.ls!(path) != []
```

Comment:
Maybe we want to check only if there have been files in the last 60 minutes instead?

Reply:
Since we do this behind the scenes I think it's a nice way of informing the user, especially that with no sessions we have the space for this. wdyt?

Reply:
You are right, let's always keep it.